src/pages/sym.mdx

```diff
@@ -0,0 +1,132 @@
+Semicolons cannot be escaped, as they are not valid in any part of entries.
+Comments should be ignored by the implementation.
+
+After removing the comment (if any), the line shall be split into non-empty **tokens**, separated by one or more consecutive **whitespace** characters.
+ASCII spaces (` `, U+0020) must be treated as whitespace; implementations may also treat tabs (`\t`, U+0009) as whitespace, but this is not portable to some older implementations.
+Other Unicode "White Space" characters must *not* be treated as whitespace.
+Leading or trailing whitespace must be ignored.
+
+### Line processing
+
+If any part of a line cannot be processed by the implementation, it (and possibly the whole line) shall be ignored, and a warning should be produced.
+
+A line without any tokens shall be silently ignored, to permit empty lines, even if they contain whitespace and/or a comment.
+
+A line with only one token shall be ignored, being reserved for future extensions.
+Encountering one may produce a warning.
+
+Otherwise, the line defines a single **symbol**: the first token provides its location, the second token provides its name, and further tokens (if any) may provide extra metadata.
+Extra tokens are currently not specified, but may be so in the future; so, implementations should ignore them, and may produce a warning in doing so.
+
+#### Symbols
+
+A symbol is uniquely defined by the combination of its location and its name; symbol names are case-sensitive.
+If the same symbol (identical name *and* identical location) is defined twice, one of the two definitions must be silently ignored.
+
+If two distinct symbols are defined with the same name (but different locations), neither definition shall be ignored, and this should not be an error.
+A warning may be produced, though not necessarily when loading the sym file: for example, "displaying names instead of memory addresses" accomodates this case just fine, however a "go to symbol" feature wouldn't, and a warning could be produced when the user tries to go to an ambiguous target.
+
+##### Location
+
+The location shall be of the form `<bank>:<address>`, both `<bank>` and `<address>` being case-insensitive hexadecimal numbers, possibly zero-padded, separated by a colon (`:`, U+003A).
+Typical hexadecimal prefixes (`0x`, `$`, etc.) are not allowed.
+An implementation may also allow the bank to be the case-insensitive keyword `BOOT` to specify symbols in the loaded boot ROM.
+If the bank is omitted, then the colon must be as well; the results of such lines are implementation-defined.
+
+The address must be in 16-bit range (0 to $FFFF inclusive), the bank must be in 32-bit range (0 to $FFFFFFFF inclusive); implementations may abandon processing the line if either number is larger than these limits.
+Implementations may accept bank numbers higher than $FFFFFFFF.
+Nonsensical locations (e.g. `2:9800`) may produce a warning.
```

Comment:
Don't hedge on the boot bank (just allow it). Also, don't mistake supporting an address format for accepting the symbol definition.
I'd honestly rewrite the whole thing. There's so many fuzzy requirements here that nobody could possibly hope to know what's valid and what's not. Let me be clear about something: the whole point of a specification is to define behaviors. If existing behaviors are inconsistent, they'll have to adapt. There's _no way_ to write a specification that is useful and accomodates all existing implementations from the past 20+ years; any attempt at the former will have to forgo the latter to some extent. Existing implementations can adapt if the changes are minimal.

"Locations can have one of the following three forms:

- `<number>:<number>`, indicating a location by its bank and address;
- `BOOT:<number>`, indicating a location in the boot ROM (the GB's internal monitor ROM or any emulator stand-in); or
- `<number>`, indicating a location by its address irrespective of bank (i.e., a location in every possible bank).

All forms are case-insensitive; in all cases, `<number>` is a sequence of one to four hexadecimal digits, without any prefixes indicating the base (like `0x` or `$`). Implementations MAY ignore symbols defined with locations in the last two forms if they don't support those types of addresses. Implementations MAY issue a warning if a location is nonsensical for the chosen hardware configuration, such as  `2:9800` (or `1:9800` in DMG mode)."

Note that I removed banks greater than 16 bits. There's exactly zero use cases for them. If some implementation supports garbage there, great, but there's no point putting it on a spec.